Makefile

```diff
@@ -76,7 +76,7 @@ clean: bitbake-clean
 	rm -f .docker-image
 
 .PHONY: conf
-conf: ${POKY}/build/conf/local.conf ${POKY}/build/conf/bblayers.conf
+conf: .docker-image ${POKY}/build/conf/local.conf ${POKY}/build/conf/bblayers.conf
```

Comment:
Since this target doesn't use docker_run, perhaps .docker-image should be added to the `${POKY}/build/conf/%:` target below instead?

Reply:
I've moved the .docker-image as suggested and re-tested. Both ways have the same end effect of building the docker image before using it.